include/core/ib_particle.h

```diff
@@ -85,47 +85,69 @@ class IBParticle
 
   // This class defines values related to a particle used in the sharp interface
   // IB. Each particle defined will have these value used in the solver.
-  Point<dim>     position;
-  Point<dim>     last_position;
-  Tensor<1, dim> forces;
-  Tensor<1, dim> last_forces;
-  unsigned int   particle_id;
+  Point<dim>              position;
+  std::vector<Point<dim>> last_position;
+  Tensor<1, dim>          forces;
+  Tensor<1, dim>          last_forces;
```

Comment:
```suggestion
  Tensor<1, dim>          force;
  Tensor<1, dim>          last_force;
```
Singular also no?
What is last force, is it the force at the previous iteration? I'd use previous in this case to be clearer.